### ISSUE TITLE
Allow to customize crash log level

### ISSUE THREAD

**Issue by fleficher** [DataDog/dd-sdk-android#751]
**Is your feature request related to a problem? Please describe.**
We should be able to change the default log level for crashes. Currently, it's always set to `EMERGENCY` level while we would like to map them to `ERROR` level.

**Describe the solution you'd like**
From the Datadog `Configuration.Builder`, we should be able to customize the crash log level as:

```kotlin
 Datadog.initialize(
            context = context,
            configuration = Configuration.Builder(
                crashReportsEnabled = true,
                crashLogPriority = Log.ERROR,
            )
           // ...
        )
```

**Comment by xgouchet** [comment]
Hi @fleficher , thanks a lot for this proposal. Right now we purposefully decided to mark crashes as Emergencies to distinguish them from manually logged errors (from `catch` blocks, for instance). 
Out of curiosity, what's your need into marking those as `ERROR` instead of `EMERGENCY`?

**Comment by fleficher** [comment]
@xgouchet Currently we're using only 5 levels of logs (error, warning, etc.), which make it simplier to make dashboards.
For the moment, we've made a pipeline to change the log level for crashes; which is working well.

**Comment by StylianosGakis** [comment]
We're having the exact same case. Are there any plans to have this be configurable? No other service we got is using the Emergency level log, which will make ours stand out and could also possibly make some people worried when those numbers start going up.

**Comment by xgouchet** [comment]
Hi @StylianosGakis , this is not in our immediate roadmap for now, as we want to distinguish actual crash from simply errors reported from `try/catch` blocks. You can use a logs pipeline to map the `emergency` status to something else, like @fleficher mentionned.

**Comment by StylianosGakis** [comment]
Yeah in our case it was mostly to not worry all the other engineers about suddenly having emergency logs while they've had 0 for a long time now.
I understand that this is not in the roadmap right now, and I will try to simply inform everyone in my company instead to understand why they are there. Thanks for the response 🤗
